app/components/sidebars/main/channels_list/list/index.js

```diff
@@ -50,7 +50,6 @@ function mapStateToProps(state) {
         sidebarPrefs.sorting,
         true, // The mobile app should always display the Unreads section regardless of user settings (MM-13420)
         sidebarPrefs.favorite_at_top === 'true' && favoriteChannelIds.length,
-        collapsedThreadsEnabled,
```

Comment:
?

Reply:
We were sending this to add "Threads" to the sections. As we don't need it anymore, I have removed that code.

Reply:
I see this prop being used in the list component, please double check